SingularityService/src/main/java/com/hubspot/singularity/config/UIConfiguration.java

```diff
@@ -21,6 +25,9 @@
   @JsonProperty
   private String baseUrl;
 
+  @JsonProperty
+  private List<HashMap<String, String>> customQuickLinks = new LinkedList<>();
```

Comment:
i'd consider renaming this `taskQuickLinks`. Also better to use `Collections.emptyList()` as opposed to allocating an entirely new linked list.